packages/admin-stories/src/admin/snackbar/CustomSnackbar.tsx

```diff
@@ -0,0 +1,78 @@
+import { SnackbarProvider, useSnackbarApi } from "@comet/admin";
+import { Button, List, ListItem, Snackbar } from "@material-ui/core";
+import { storiesOf } from "@storybook/react";
+import * as React from "react";
+import styled from "styled-components";
+
+const ShowSnackbarButton = styled(Button)`
```

Comment:
In my opinion, the stories should be as minimal as possible, while showing everything that is important. The color of the buttons is not important and should be a) removed entirely (use `color=primary` and `color=secondary`) or b) moved to the bottom of the story.